frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardGraph.tsx

```diff
@@ -43,4 +55,11 @@ const MonitoringDashboardGraph: React.FC<MonitoringDashboardGraphProps> = ({
   );
 };
 
-export default MonitoringDashboardGraph;
+const mapDispatchToProps = (dispatch): DispatchProps => ({
+  patchQuery: (v: QueryObj) => dispatch(queryBrowserPatchQuery(0, v)),
```

Comment:
This seems very odd to me that every graph is using the same index.

@kyoto can you please provide insight into the proper usage.

Reply:
Always using the index `0` looks wrong to me, so I'm not sure why this appears to be working.

For the admin perspective, I used `onMouseEnter` as a workaround for the problem (see https://github.com/openshift/console/pull/4060).

The root problem is that the tooltips assume the query is in Redux, which is actually only true for the Metrics page. Longer term, we should aim to fix that in the tooltips and remove these workarounds.

Reply:
Thanks @kyoto 
Longer term we will move to share more of the dashboards.